src/components/Popover/PopoverExample.tsx

```diff
@@ -0,0 +1,149 @@
+import clsx from 'clsx';
+import React, { ReactNode, useState, useRef, MutableRefObject } from 'react';
+import { Popover } from './Popover';
+import { Button } from '../Button/Button';
+import { Heading } from '../Heading/Heading';
+import { NotificationList } from '../NotificationList/NotificationList';
+import { NotificationListItem } from '../NotificationListItem/NotificationListItem';
+import { PopoverBody } from '../PopoverBody/PopoverBody';
+import { PopoverHeader } from '../PopoverHeader/PopoverHeader';
+import { TextPassage } from '../TextPassage/TextPassage';
+
+export interface Props {
+  /**
+   * Child node(s) that can be nested inside component
+   */
+  children: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Size prop to pass into Popover
+   */
+  size?: 'sm' | 'lg' | 'xl';
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const PopoverExample: React.FC<Props> = ({
+  children,
+  className,
+  size,
+  ...other
+}) => {
+  const [popoverOpen, setPopoverOpen] = useState(false);
+  const popoverButton = useRef() as MutableRefObject<HTMLSpanElement>;
+
+  function openContinuePopover() {
+    setPopoverOpen(true);
+  }
+
+  function closeContinuePopover(event: any) {
+    setTimeout(() => {
+      popoverButton.current.focus();
+    }, 1);
+    if (event) {
+      event.preventDefault();
+    }
+    setPopoverOpen(false);
+  }
+  const componentClassName = clsx('tooltip-popover', className, {});
+  return (
+    <div
+      style={{
+        padding: '1rem',
```

Comment:
What's the context around this inline styling? Just for display?

Reply:
yes, and I replaced it with another <header> wrapper instead, since this subsection is essentially identical to the first subsection